classes/field/localizer.php

```diff
@@ -104,14 +104,21 @@ public static function localize_cb(){
 	 * @return array
 	 */
 	protected static function error_strings(){
-		$strings = array(
-			'mixed_protocol' => __( 'Submission URL and current URL protocols do not match. Form may not function properly.', 'caldera-forms' ),
-			'jquery_old'     => __( 'An out of date version of jQuery is loaded on the page. Form may not function properly.', 'caldera-forms' )
-		);
-
-		if( ! current_user_can( Caldera_Forms::get_manage_cap( 'admin' )) ){
-			unset( $strings[ 'jquery_old' ] );
-		}
+        if( current_user_can( Caldera_Forms::get_manage_cap('admin') ) ) {
```

Comment:
@New0 This is a bit of a change - less info for people who can't fix the problem, more info for people who can.